### DISCUSSION TITLE
Difference between skipping and ending Pomodoro session

### DISCUSSION THREAD

**Discussion by bytrangle** [super-productivity/super-productivity#2584] [Q&A]
Sometimes I would mark a task as completed before the current Pomodoro session runs out. I would like to start a new task and reset the Pomodoro to the original duration. Should I click "Skip Pomodoro session" or "End Pomodoro session" and what is the difference between these two?

**Comment by johannesjo** [comment]
Hi there! How does your pomodoro configuration look like and what do you mean by original duration?

**Reply by bytrangle** [reply]
*Original duration* is the duration of work sessions, e.g. 30 minutes. If I hit the timer, it counts down from 30 minutes. Suppose I finish the task after 25 minutes, so I stop the timer and mark the task as complete. So the timer is stuck at 5 minutes and the Pomodoro session is effectively not finished yet. Hope that makes sense.

I wonder whether I should "Skip Pomodoro" or "End Pomodoro"? It is not that of a big deal, but I'd like to know :).

**Reply by johannesjo** [reply]
Hm I think that's not how pomodoro is supposed to work. You gotta take your breaks :D 

There are long breaks and short breaks in pomodoro. Long breaks occur after X sessions. If you use end session all is reset to the first cycle. Skip ends to the current cycle and skipps to the next. So the wording of "end session" is misleading. I'll adjust it to something else.

Thanks for bringing this to my attention :)
